### PR TITLE
Reworked Checklist for each PR

### PR DESCRIPTION
**Pull request by ThomasNaderBMW**
#### Reference to a related issue in the repository
None.

#### Add a description
Checklist reworked as decided in the Q4/2020-Meeting at ASAM (27.01.2021).

**Some questions to ask**:
What is this change?
The checklist inside each PR to tell how "grown up" your PR is.
I also deleted the "Mention a member"-part as this is done by assigning reviewers (agree on that?)
What does it fix?
PR-Improvement.
Is this a bug fix or a feature? Does it break any existing functionality or force me to update to a new version?
PR-Improvement.
How has it been tested?
The test is done by this PR.
See the new content from here below:

#### Take this checklist as orientation for yourself, if this PR is ready for the Change Control Board:
- [x] My suggestion follows the [style and contributors guidelines](https://opensimulationinterface.github.io/osi-documentation/open-simulation-interface/doc/howtocontribute.html).
- [x] I have taken care about the [documentation](https://opensimulationinterface.github.io/osi-documentation/open-simulation-interface/doc/commenting.html).
- [x] I have done the [DCO signoff](https://opensimulationinterface.github.io/osi-documentation/open-simulation-interface/doc/howtocontribute.html#developer-certification-of-origin-dco).
- [x] My changes generate no errors when passing CI tests. 
- [x] I have successfully implemented and tested my fix/feature locally.
- [x] Appropriate reviewer(s) are assigned.

If you can’t check all of them, please explain why.
If all boxes are checked or commented and you achieved at least one positive review, you can assign the label readyForCCB.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -10,14 +10,13 @@ What does it fix?
...
-
-- [ ] My code and comments follow the [style guidelines](https://opensimulationinterface.github.io/osi-documentation/osi/commenting.html) and [contributors guidelines](https://opensimulationinterface.github.io/osi-documentation/osi/howtocontribute.html) of this project.
-- [ ] I have performed a self-review of my own code.
-- [ ] I have made corresponding changes to the [documentation](https://github.com/OpenSimulationInterface/osi-documentation).
-- [ ] My changes generate no new warnings.
-- [ ] I have added tests that prove my fix is effective or that my feature works.
-- [ ] New and existing unit tests / travis ci pass locally with my changes.
\ No newline at end of file
+If you can’t check all of them, please explain why.
+If all boxes are checked or commented and you achieved at least one positive review, you can assign the label readyForCCB!
```
Comment (kmeids) [inline .github/pull_request_template.md:22]: @ThomasNaderBMW, some minor changes
If all boxes are checked or commented and you **have** achieved at least one positive review, you can assign the label **ReadyForCCBReview!**

Comment (ThomasNaderBMW) [reply .github/pull_request_template.md]: Thank you for the correction @kmeids ! Of course, totally agree.

**Review by kmeids — APPROVED** [review]
Approved during CCB meeting 11.02.2021

**Comment by kmeids** [comment]
**Output of CCB meeting 11.02.2021**
Actions:
1. @pmai to merge please.

**Pull request merged**